### PR TITLE
Fix link with changing compositor-only properties

### PR DESCRIPTION
**Pull request by pathliving**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

File: src/site/content/en/blog/rendering-performance/index.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ browser jumps to just do compositing.
 This final version is the cheapest and most desirable for high pressure points
 in an app's lifecycle, like animations or scrolling.
 
-Note: If you want to know which of the three versions above changing any given CSS property will trigger head to [CSS Triggers](https://csstriggers.com). And if you want the fast track to high performance animations, read the section on [changing compositor-only properties](stick-to-compositor-only-properties-and-manage-layer-count).
+Note: If you want to know which of the three versions above changing any given CSS property will trigger head to [CSS Triggers](https://csstriggers.com). And if you want the fast track to high performance animations, read the section on [changing compositor-only properties](https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count).
```
Comment (malchata) [inline src/site/content/en/blog/rendering-performance/index.md:110]:
```suggestion
Note: If you want to know which of the three versions above changing any given CSS property will trigger head to [CSS Triggers](https://csstriggers.com). And if you want the fast track to high performance animations, read the section on [changing compositor-only properties](https://web.dev/stick-to-compositor-only-properties-and-manage-layer-count/).
```

Comment (malchata) [reply src/site/content/en/blog/rendering-performance/index.md:110]: Good catch on this. However, we'd want to redirect to the web.dev version, as we're migrating old content from developers.google.com to web.dev and developer.chrome.com.

**Comment by malchata** [comment]
@pathliving, you'll need to sign the CLA before I can merge this change. Otherwise, everything looks good to me.

**Comment by pathliving** [comment]
[malchata](https://github.com/malchata) thank you, done

File: src/site/content/en/blog/rendering-performance/index.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ browser jumps to just do compositing.
 This final version is the cheapest and most desirable for high pressure points
 in an app's lifecycle, like animations or scrolling.
 
-Note: If you want to know which of the three versions above changing any given CSS property will trigger head to [CSS Triggers](https://csstriggers.com). And if you want the fast track to high performance animations, read the section on [changing compositor-only properties](stick-to-compositor-only-properties-and-manage-layer-count).
+Note: If you want to know which of the three versions above changing any given CSS property will trigger head to [CSS Triggers](https://csstriggers.com). And if you want the fast track to high performance animations, read the section on [changing compositor-only properties](https://web.dev/stick-to-compositor-only-properties-and-manage-layer-count/).
```
Comment (malchata) [inline src/site/content/en/blog/rendering-performance/index.md:110]:
```suggestion
Note: If you want to know which of the three versions above changing any given CSS property will trigger head to [CSS Triggers](https://csstriggers.com). And if you want the fast track to high performance animations, read the section on [changing compositor-only properties](/stick-to-compositor-only-properties-and-manage-layer-count/).
```

**Pull request merged**